inc/admin/class-admin.php

```diff
@@ -146,20 +146,21 @@ public function check_for_event_submissions(){
 				$parent_id = '';
 				$rootCategories = array();
 				$rootTerms = get_terms( array( 'taxonomy' => $taxonomyActive, 'parent' => 0, 'hide_empty' => false ) );
-				foreach ($rootTerms as $rootTerm) {
-					$rootCategories[] = array('id' => $rootTerm->term_id, 'name' => $rootTerm->name);
-				}
-				
+				//use wp_list_pluck for cleaner code.
+				$rootCategories = wp_list_pluck($rootTerms, 'term_id','name');
+
 				foreach($lines as $line){
-					$split_line = explode('/', $line);
+					//use str_getcsv to enable eclosure with quotes.
+					$split_line = str_getcsv(stripslashes($line), '/','"');
```

Comment:
usging PHP built-in function to strip lines into arrays makes for much easier/faster code development, it also comes with the ability to enclose strings in quotes as the original bug reported.